web/concrete/src/Application/Application.php

```diff
@@ -58,9 +58,9 @@ public function shutdown($options = array())
             }
 
             $logger = new Logger();
-            $r = Request::getInstance();
+            $r = $this['request']->getInstance();
```

Comment:
I wondered about that, but I only tested it by calling `getClientIp` and when that worked I thought it's okay.. Guess it's not. Thanks for your detailed answer